### PR TITLE
Update docs for CDN tracking

### PR DESCRIPTION
**Pull request by heff**

### CODE REVIEW

File: source/index.html.md [hunk 1]
```diff
@@ -167,6 +169,7 @@ video_duration | The length of the video in **seconds** [float]
 video_stream_type | 'live' or 'on-demand'
 video_producer | The producer of the video title
 video_encoding_variant | An optional detail that allows you to compare different encoding settings.
+video_cdn | The name of the CDN, for comparing across CDN providers. Possible values include 'akamai', 'fastly', 'cloudfront', 'limelight', 'level3', 'custom', 'none'. Contact mux support to include additional CDN providers.
```
Comment (heff) [inline source/index.html.md:172]: @jondahl let me know if you want to discuss what values belong here. I don't really feel like s3 belongs. GCS is a weird one that could be included.

**Comment by mmcc** [comment]
:shipit:

File: source/index.html.md (hunk 1 above)
Comment (heff) [inline source/index.html.md:172]: @jondahl just realized we still need to pull this into the docs. Want to review first or does it look good?

**Comment by mmcc** [comment]
@heff what's this?

**Comment by heff** [comment]
Apparently a docs change that's been waiting on Jon to review for 8 months. :) @jsanford8 do these details still apply? If so we should pull it in.

**Comment by jsanford8** [comment]
Nope, we've already pulled this in. You can open a new one if you want to change the description, but it's in there now.

**Pull request closed without merging**